runtime/shared_common/shrinit.cpp

```diff
@@ -3276,8 +3303,11 @@ j9shr_init(J9JavaVM *vm, UDATA loadFlags, UDATA* nonfatal)
 	if (parseResult==RESULT_DO_PRINTSTATS ||
 		parseResult==RESULT_DO_PRINTALLSTATS ||
```

Comment:
might as well fix the formatting for all of the conditions

Reply:
Fixed